docs/OTSession.md

```diff
@@ -57,4 +57,6 @@ class App extends Component {
   
   * **streamCreated** (Object) — Sent when a new stream is created in this session.
 
-  * **streamDestroyed** (Object) - Sent when a stream is no longer published to the session.
\ No newline at end of file
+  * **streamDestroyed** (Object) - Sent when a stream is no longer published to the session.
+  
+  * **streamPropertyChanged** (Object) - Sent when a stream's properties have changed, such as video or audio becoming available or unavailable.
```

Comment:
How about something like this
`Sent when a stream has started or stopped publishing audio or video or if the video dimensions of the stream have changed`?

Unavailable makes it seem more like an error IMHO.

Reply:
Yeah, that's more accurate. Done.